examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,91 @@
+package gnopages
+
+func init() {
+	path := "contribute"
+	title := "Contributor Ecosystem: Call for Contributions"
+	// XXX: description := "Be a an early adopter and contributor to gno.land through bounties, grants, and Game of Realm's notoable contributions network."
+	body := `
+
+gno.land puts at the center of its identity the contributors that help to create and shape the project into what it is; incentivizing those who contribute the most and help advance its vision. Eventually, contributions will be incentivized directly on-chain; in the meantime, this page serves to illustrate our current off-chain initiatives.
+
+gno.land is still in full-steam development. For now, we're looking for the earliest of adopters; curious to explore a new way to build smart contracts and eager to make an impact. Joining gno.land's development now means you can help to shape the base of its development ecosystem, which will pave the way for the next generation of blockchain programming.
+
+If you are interested in contributing to gno.land, you can jump on in on our [GitHub monorepo](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md) - where most development happens. Additionally, you can look out to help on specific issues labeled as bounties. All contributions will then concur to form your profile for Game of Realms. In the following sections, we delve into these initiatives in a bit more detail.
+
+## Gno Bounties
+
+The Gno bounty program is a good way to familiarize yourself with the project, and how Gno code works. We will maintain open and rewardable bounties in the gnolang/gno repository, and you can search all available bounties by using the ["bounty" label](https://github.com/gnolang/gno/labels/bounty).
+
+Recommendations on participating in the gno.land Bounty Program:
+
+- Identify the bounty you want to work on, and first start working in your local environment.
+- Once you have spent time on the code related to the bounty, we recommend submitting a 'draft' PR as soon as possible.
+    - The draft PR doesn't indicate that the bounty has been assigned to you, others are free to work on other draft PRs for the bounty.
+    - After submitting the 'draft' PR, continue working until you are ready to mark the PR as "ready for review".
+    - The core team will review the bounty PR submission, and determine if it qualifies for the bounty reward.
+- Ask for clarification early if an element on the DoD or implementation design is unclear.
+    - Aside from posting the issue as a bounty, keeping regular updates with the core team on the bounty being undertaken is key to being on the right track.
+    - The Definition of Done requires adhesion to the [contributing guidelines](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md); additionally, it is expected that any newly added code or functionality is properly documented, tested and covered, at least in 80% of added code.
```

Comment:
```suggestion
    - The Definition of Done requires adhesion to the [contributing guidelines](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md); additionally, it is expected that any newly added code or functionality is properly documented, tested and covered, at least in 80% of added code.
```

what about not specifying anything and making sure tht CONTRIBUTING.md is enough?

Reply:
(speaking about "additionally, it is expected...")

Reply:
also not making sense for non-code PRs.

Reply:
I think it's an important point to add. We can add this to CONTRIBUTING.md as well; but we should have this as a point that is stressed out in capital letters rather than being "fine print". (Ie. we should not give bounties to people who don't make efforts to document and test).

For what concerns non-code PRs; there is no newly added code, so this section wouldn't apply?

I would expect most bounties to be related to code anyway; if not at the early bounty (creating a proposal for the anti-squatting system, for instance) for a later bounty (creating the implementation)